### PR TITLE
[tests/README.md] Update information missing about Selenium Server and VNC Viewer

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Include information missing about Selenium Server and VNC Viewer. How to view tests happen in realtime with VNC Viewer.

#### Links to related tickets (GitHub, Redmine, ...)

* https://github.com/aces/Loris/issues/5857#issuecomment-566162670

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -48,6 +48,25 @@ npm run lint:javascript
   
 ## Advanced Workflow
 
+#### View tests performing in realtime
+You can view the testing process as it happens in realtime with [Selenium Server](https://selenium.dev/downloads/) and using [VNC Viewer](https://www.realvnc.com/en/connect/download/viewer/).
+
+Such as an example workflow:
```
Comment (johnsaigle) [inline test/README.md]:
```suggestion
```

File: test/README.md [hunk 2]
```diff
@@ -48,6 +48,25 @@ npm run lint:javascript
...
+
+1) Start the Selenium Server:
+    ```
+    java -jar selenium-server-standalone-2.53.1.jar
```
Comment (johnsaigle) [inline test/README.md]: Any idea whether this version is current and consistent with what Docker is using?

File: test/README.md [hunk 3]
```diff
@@ -48,6 +48,25 @@ npm run lint:javascript
   
 ## Advanced Workflow
 
+#### View tests performing in realtime
```
Comment (johnsaigle) [inline test/README.md]:
```suggestion
#### View tests in realtime
```

File: test/README.md [hunk 4]
```diff
@@ -48,6 +48,25 @@ npm run lint:javascript
...
+You can view the testing process as it happens in realtime with [Selenium Server](https://selenium.dev/downloads/) and using [VNC Viewer](https://www.realvnc.com/en/connect/download/viewer/).
```
Comment (johnsaigle) [inline test/README.md]:
```suggestion
You can view the testing process as it happens with [Selenium Server](https://selenium.dev/downloads/) and using [VNC Viewer](https://www.realvnc.com/en/connect/download/viewer/).
```

File: test/README.md (hunk 2 above)
Comment (driusan) [reply test/README.md]: There's definitely at least 3.x released, but I don't know what Docker is using.

Comment (PapillonMcGill) [reply test/README.md]: Any info about `java` requirement?
Oracle JDK or OpenJDK?
min Java version?

Comment (kongtiaowang) [reply test/README.md]: Docker is running with selenium 3.8.1-aluminum

Comment (kongtiaowang) [reply test/README.md]: 1) Start the Selenium Server:     This line is unnecessary, The docker test has included the selenium server.

**Pull request merged**